explainer.md

```diff
@@ -378,35 +397,31 @@ this aspect.
     </td>
   </tr>
   <tr>
-    <td>onpromptdismiss onpromptaction</td>
-    <td>Event handlers as discussed above.</td>
-  </tr>
-  <tr>
-    <td>iconstyle</td>
+    <td>type-ext</td>
     <td>
-      Used to allow the site to select different icon styles. Supported values:
-      <table>
-        <tr>
-          <td>solid</td>
-          <td>
-            <img src="images/image12.png" />
-          </td>
-        </tr>
-        <tr>
-          <td>outline</td>
-          <td>
-            <img src="images/image13.png" />
-          </td>
-        </tr>
-        <tr>
-          <td>none</td>
-          <td>
-            <img src="images/image14.png" />
-          </td>
-        </tr>
-      </table>
+      Allows specifying additional parameters for certain permission types, in
+      the form of space-separate key: value pairs.
+      The supported key/values are:
+      <ul>
+        <li><code>sysex: true/false</code> (for the 
+         <a href="https://webaudio.github.io/web-midi-api/#permissions-integration">
+        midi</a> permission type)
+        </li>
+        <li><code>precise:true/false</code> (for the 
+        <a href="https://www.w3.org/TR/geolocation/#position_options_interface">
```

Comment:
Are there user agents that distinguish on a per-site level between coarse/precise location permissions? To my knowledge, this distinction is only made at app-level permissions.

Reply:
I know that Chrome doesn't at the moment, but it seems like a natural next step so I would be surprised if this doesn't change. It seemed like a reasonable addition and it allows the UA to have more information when picking the text for the button. E.g. "Use precise location" vs simply "Use location"